<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GDS2WriterOptionPage</class>
 <widget class="QWidget" name="GDS2WriterOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GDS2 Writer Options</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="7" column="0" colspan="3">
       <widget class="QCheckBox" name="write_timestamps">
        <property name="text">
         <string>Write current time to time stamps (BGNLIB, BGNSTR)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max. cell name length</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>(&lt;4000 recommended, absolute limit 8191)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="cell_name_length_le"/>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="write_cell_properties">
           <property name="text">
            <string>cell</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="write_file_properties">
           <property name="text">
            <string>layout</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>properties (at the cost of compatibility)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="max_vertex_le"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>(keep empty for unspecified limit)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QCheckBox" name="multi_xy_cbx">
        <property name="text">
         <string>Multi-XY record mode for boundaries
(enables infinitely large polygons/paths at the cost of compatibility)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Library name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Max. vertices</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="libname_le"/>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QCheckBox" name="no_zero_length_paths">
        <property name="text">
         <string>Eliminate zero-length paths (convert to BOUNDARY)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QCheckBox" name="resolve_skew_arrays_cbx">
        <property name="text">
         <string>Resolve skew (non-orthogonal) arrays into single instances</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>524</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>libname_le</tabstop>
  <tabstop>cell_name_length_le</tabstop>
  <tabstop>max_vertex_le</tabstop>
  <tabstop>multi_xy_cbx</tabstop>
  <tabstop>no_zero_length_paths</tabstop>
  <tabstop>write_timestamps</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
